P4-grpc-cve-process.md

```diff
@@ -0,0 +1,97 @@
+* You are unsure how a vulnerability affects gRPC
+* You think you discovered a vulnerability in another project that gRPC depends on.
+
+#### When should I not report a vulnerability
+* You need help tuning gRPC security
+* You need help applying security related updates
+* Your issue is not security related
+
+#### Security Vulnerability Response
+Each report is acknowledged and analyzed by Product Security Team members within 3 working days. 
+
+Any vulnerability information shared with Product Security Team stays within the gRPC project and will not be disseminated to other projects unless it is necessary to get the issue fixed.
+
+As the security issue moves from triage, to identified fix, to release planning we will keep the reporter updated.
+
+
+#### Public Disclosure Timing
+A public disclosure date is negotiated by the gRPC product security team and the bug submitter. We prefer to fully disclose the bug as soon as possible once a user mitigation is available. It is reasonable to delay disclosure when the bug or the fix is not yet fully understood, the solution is not well-tested, or for vendor coordination. The timeframe for disclosure is from immediate (especially if it’s already publicly known) to a few weeks. As a basic default, we expect report date to disclosure date to be on the order of 7 days. The gRPC product security team holds the final say when setting a disclosure date.
+
+### Public Disclosure Process
+
+If you know a publicly disclosed vulnerability, please IMMEDIATELY email grpc-security@googlegroups.com to inform the Product Security Team (PST) about the vulnerability, so that we may start a patch release and communication process.
+
+## Patch, Release, and Public Communication
+
+For each vulnerability, a member of the Product Security Team (PST) will volunteer to lead the coordination in fixing, releasing and communicating on the disclosure to the rest of the community. This lead is referred to as the Fix Lead.
+* The fix lead will make an initial determination on whether the issue could be a vulnerability. At this stage they may go back to the reporter for more information.
+* The fix lead will work to identify relevant engineers (the Fix Team) and CC them onto the disclosure thread for further triage of the issue. This should be completed within the first 24 hours of Disclosure.
+* The triage of the issue would involve, at first, to reproduce and/or evaluate the report to see whether this is a real vulnerability.
+* The fix will be worked on privately, and will land once there are approvals on the commits, and it has been fully validated to address the issue.
+
+## Fix Development Process
+
+These steps should be completed within the 1-7 days of Disclosure.
+
+* The Fix Lead and the Fix Team will create a CVSS using the CVSS Calculator. They will also determine the effect and severity of the bug. The Fix Lead makes the final call on the calculated risk; it is better to move quickly than make the perfect assessment.
```

Comment:
Link to CVSS (https://www.first.org/cvss/specification-document) and calculator https://www.first.org/cvss/calculator/3.0